src/api/files.js

```diff
@@ -0,0 +1,25 @@
+'use strict'
+
+const argCommand = require('../cmd-helpers').argCommand
+
+module.exports = send => {
+  return {
+    cp: argCommand(send, 'files/cp'),
+    ls: argCommand(send, 'files/ls'),
+    mkdir: argCommand(send, 'files/mkdir'),
+    stat: argCommand(send, 'files/stat'),
+    rm: function (path, opts, cb) {
```

Comment:
Was in the process of making this and also pulling latest changes from master to feat/files-api and realised that 0.4.0 doesn't know about ndjson stuff yet, which breaks the tests, will wait for that to land

https://github.com/ipfs/go-ipfs/pull/1381#issuecomment-165997124